src/main/java/com/netflix/imflibrary/st0377/header/GenericPictureEssenceDescriptor.java

```diff
@@ -50,6 +50,7 @@
     public static final String componentMaxRefUL = "urn:smpte:ul:060e2b34.01010105.04010503.0b000000";
     public static final String blackRefLevelUL = "urn:smpte:ul:060e2b34.01010101.04010503.03000000";
     public static final String whiteRefLevelUL = "urn:smpte:ul:060e2b34.01010101.04010503.04000000";
+    public static final String componentDepthUL = "urn:smpte:ul:060e2b34.01010102.04010503.0a000000";
```

Comment:
How is `componentDepthUL` related to TSP 2121-1?

Reply:
I added it into a separated commit: https://github.com/Netflix/photon/pull/234/commits/4e2c1f861905d57385af6e40b2c2096b461e94c7

It's required to get the component depth for the Descriptor stored in MXF, [pixelBitDepth](https://github.com/Netflix/photon/blob/4e2c1f861905d57385af6e40b2c2096b461e94c7/src/main/java/com/netflix/imflibrary/st2067_2/CompositionImageEssenceDescriptorModel.java#L98) is deduced from the color model.

In TSP App2121 validation, it's required as I haven't subDescriptor so I added [these lines](https://github.com/Netflix/photon/blob/4a4f1613e9f13c5a1ca667deecc105e4c1dfd112/src/main/java/com/netflix/imflibrary/st2067_2/CompositionImageEssenceDescriptorModel.java#L222) to get a correct bit depth. Does I have made something wrong ?

Reply:
I am surprised it is not already there since the same constraints apply to ST 2067-20:

_Component Depth shall be present and shall be equal to the Pixel Bit Depth used_

Reply:
I'm agreed, I can plan to add check on App2/App2e too.

Reply:
FYI, I have pushed #239 to fix that.